### ISSUE TITLE
Extra Wall description when using JW with Show no charges checked 

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#388]
Testing Dragging in [dev version 21](https://www.colorado.edu/physics/phet/dev/html/balloons-and-static-electricity/1.4.0-dev.21/phet/balloons-and-static-electricity_all_phet_debug.html?accessibility)

<img width="749" alt="screen shot 2018-04-06 at 13 41 34" src="https://user-images.githubusercontent.com/1902116/38431964-9060461e-39a0-11e8-9843-b9fc732baa28.png">

@jessegreenberg, I found this extra wall description (Show all charges description) describing the wall when jumping from the sweater to the wall with Show charge differences checked. i was able to reproduce this, too.

This only happens when using the JW keys. Regular dragging with WASD and Arrow keys seem to be fine. Other jump keys seem to be fine.

**Comment by phet-steele** [comment]
@terracoda, what is the extra description? Asking since this is in dev testing, so QA should know what issues are being found. I am not at all familiar with the descriptions, so I don't know what is "extra" in the above image.

**Comment by phet-steele** [comment]
For phetsims/QA/issues/112.

**Comment by terracoda** [comment]
Thanks @phet-steele, the extra description is:

> Wall has  many pairs of negative and positive charges.

That part is the wall's description when _Show all charges_ is checked.

**Comment by phet-steele** [comment]
@terracoda Thanks!

**Comment by jessegreenberg** [comment]
Should be fixed with the above commit, @terracoda could you please verify with phettest?

**Comment by terracoda** [comment]
Sounds beautiful! Nice work @jessegreenberg! 
I am removing the pending-testing label and closing this issue.
